yarn-project/end-to-end/src/e2e_inclusion_proofs_contract.test.ts

```diff
@@ -72,20 +72,39 @@ describe('e2e_inclusion_proofs_contract', () => {
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 55 to 55]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This could be made a bit easier to follow if using an additional describe and having the steps individually in there. It is doing multiple things in this test so might just be nice to have it laid out easier to follow. <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4375?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->